cmd/src/server.rs

```diff
@@ -473,6 +473,17 @@ fn check_system_config(config: &TiKvConfig) {
             "err" => %e
         );
     }
+    // Check blob file dir is empty when titan is disabled
+    if !config.rocksdb.titan.enabled {
+        let titan_dir = if config.rocksdb.titan.dirname.is_empty() {
+            "titandb"
+        } else {
+            config.rocksdb.titan.dirname.as_str()
+        };
+        if let Err(e) = tikv_util::config::check_data_dir_empty(titan_dir, "blob") {
+            fatal!("check: titandb-data-dir-empty; err: {}", e);
```

Comment:
Shall we '`set_panic_mark` to avoid tikv to restart in this case? @Connor1996

Reply:
No need, or it's hard for users to run it normally later

Reply:
It's better to give a more constructive suggestion, like "titandb data dir is not empty, having enabled Titan before? if you want to disable Titan, please do ...."(you know what I mean)"